src/lookupDataProvider.js

```diff
@@ -7,6 +7,12 @@ class LookupDataProvider extends Component {
         lookupData: null
     }
 
+    async componentDidMount() {
+        const data = await lookupTablesService.getAllLookupData();
+        this.setState({
+            lookupData: data.data
+        });
```

Comment:
mozno by stacilo to mat len takto, zjednodusene:

```suggestion
        this.setState(data.data);
```

Reply:
ovela lepsie